### PR TITLE
MUSARK-794

### PR DESCRIPTION
**Pull request by jarlah**
barcode scanning improvements

### CODE REVIEW

File: src/modules/app/AppComponent.js [hunk 1]
```diff
@@ -156,20 +177,72 @@ export class App extends Component {
...
           {'Build number: ' + this.props.appSession.getBuildNumber()}
         </footer>
+
+        <Modal show={this.props.scanner.enabled && !!this.props.scanner.code} bsSize="small">
+          <Modal.Body>
+            <Form inline>
+              <FormGroup controlId="barCode">
+                <input type="text" className="form-control" readOnly value={this.props.scanner.code || ''}/>
+              </FormGroup>
+              <Button bsStyle="success" onClick={this.searchForBarCode}>Search</Button>
```
Comment (rituvesh) [inline src/modules/app/AppComponent.js]: Add language tag to support multi language.

File: src/modules/app/AppComponent.js [hunk 2]
```diff
@@ -156,20 +177,72 @@ export class App extends Component {
...
+            </Form>
+            <Loader loaded={!this.props.scanner.searchPending} />
+            {this.props.scanner.searchComplete && (!this.props.scanner.matches || this.props.scanner.matches.length === 0) &&
+              <center>No matching node or object</center>
```
Comment (rituvesh) [inline src/modules/app/AppComponent.js]: Add language tag to support multi language.

File: src/modules/app/AppComponent.js (hunk 1 above)
Comment (rituvesh) [inline src/modules/app/AppComponent.js]: Add language tag to support multi language.

**Review by steinaol — APPROVED** [review]
OK.  Vi kjører'n inn

**Pull request merged**